proto/optimint.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package optimint;
+
+// option go_package = "github.com/lazyledger/optimint/proto";
+import "tendermint/abci/types.proto";
+
+message Version {
+	uint32 block = 1;
+	uint32 app = 2;
+}
+
+message Header {
```

Comment:
General comment: I think it would be good if the fields were documented (where not obvious).